webapp/webpack/PostModal.vue

```diff
@@ -5,7 +5,8 @@
         <div class="modal-header">
           <i :class="icon"></i>
           <a :href="post.link" target="_blank">{{ time }}</a>
-          {{post.author_name}} @ {{post.source.name}}
+          <a v-if="post.author_link!=nil" :href="post.author_link">{{ post.author_name }}</a>
```

Comment:
nil non c'è in js, quindi stai confrontando con `undefined`. Puoi omettere tutto il confronto e mettere solo v-if="post.author_link". Perfetto per il resto